<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>740</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Network Topologies Generator</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>510</y>
      <width>151</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Begin Simulation</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="nodes">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>90</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
    <property name="minimum">
     <number>3</number>
    </property>
    <property name="maximum">
     <number>1000</number>
    </property>
    <property name="value">
     <number>5</number>
    </property>
   </widget>
   <widget class="QSpinBox" name="minimumDegree">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>90</y>
      <width>151</width>
      <height>31</height>
     </rect>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
    <property name="maximum">
     <number>999</number>
    </property>
    <property name="value">
     <number>2</number>
    </property>
   </widget>
   <widget class="QSpinBox" name="maximumDegree">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>90</y>
      <width>161</width>
      <height>31</height>
     </rect>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
    <property name="maximum">
     <number>999</number>
    </property>
    <property name="value">
     <number>2</number>
    </property>
   </widget>
   <widget class="QSpinBox" name="distance">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>90</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QSpinBox" name="area">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>220</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="minimum">
     <number>6</number>
    </property>
    <property name="maximum">
     <number>100000</number>
    </property>
    <property name="value">
     <number>10</number>
    </property>
   </widget>
   <widget class="QSpinBox" name="numberOfRegions">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>390</y>
      <width>151</width>
      <height>31</height>
     </rect>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
    <property name="maximum">
     <number>99999</number>
    </property>
   </widget>
   <widget class="QSpinBox" name="length">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>390</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
    <property name="maximum">
     <number>99999</number>
    </property>
    <property name="value">
     <number>2</number>
    </property>
   </widget>
   <widget class="QSpinBox" name="breadth">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>390</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
    <property name="maximum">
     <number>99999</number>
    </property>
    <property name="value">
     <number>5</number>
    </property>
   </widget>
   <widget class="QRadioButton" name="fixedRegions">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>320</y>
      <width>141</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>Fixed regions</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QRadioButton" name="fexibleRegions">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>320</y>
      <width>161</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>Flexible regions</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>220</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>Uniform</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Random</string>
     </property>
    </item>
   </widget>
   <widget class="QCheckBox" name="measures">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>450</y>
      <width>251</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>Do you want to enable Measures?</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="bc">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>490</y>
      <width>181</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>Betwenness Centrality</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="dc">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>520</y>
      <width>171</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>Degree Centrality</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="ec">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>490</y>
      <width>161</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Efficiency Centrality</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="cc">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>530</y>
      <width>161</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>Closeness Centrality</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="numberOfSimulations">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>390</y>
      <width>181</width>
      <height>31</height>
     </rect>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
    <property name="maximum">
     <number>99999</number>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>60</y>
      <width>67</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Nodes (N)</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>60</y>
      <width>151</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Minimum degree 〈δ〉min</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>60</y>
      <width>171</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Maximum degree 〈δ〉max</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>60</y>
      <width>141</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Minimum distance ( ι )</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>360</y>
      <width>181</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Number of Simulations ( φ )</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>360</y>
      <width>67</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Length</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>360</y>
      <width>67</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Breadth</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_10">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>360</y>
      <width>161</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Number of Regions ( R )</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_11">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>190</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Area ( √A )</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_12">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>190</y>
      <width>131</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Nodes distribution</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_13">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>20</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Network</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_14">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>140</y>
      <width>67</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Plane</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>160</y>
      <width>301</width>
      <height>101</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QGroupBox {
	border:1px solid #d0d0d0;

}</string>
    </property>
    <property name="title">
     <string> Waxman</string>
    </property>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>30</y>
       <width>71</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Alpha ( α )</string>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="alpha">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>60</y>
       <width>71</width>
       <height>31</height>
      </rect>
     </property>
     <property name="minimum">
      <double>0.010000000000000</double>
     </property>
     <property name="maximum">
      <double>1.000000000000000</double>
     </property>
     <property name="singleStep">
      <double>0.100000000000000</double>
     </property>
     <property name="value">
      <double>0.500000000000000</double>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>30</y>
       <width>67</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string> Beta ( β )</string>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="beta">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>60</y>
       <width>61</width>
       <height>31</height>
      </rect>
     </property>
     <property name="minimum">
      <double>0.100000000000000</double>
     </property>
     <property name="maximum">
      <double>1.000000000000000</double>
     </property>
     <property name="singleStep">
      <double>0.050000000000000</double>
     </property>
     <property name="value">
      <double>0.800000000000000</double>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="label_15">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>280</y>
      <width>67</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Region</string>
    </property>
   </widget>
   <zorder>fexibleRegions</zorder>
   <zorder>groupBox</zorder>
   <zorder>pushButton</zorder>
   <zorder>nodes</zorder>
   <zorder>minimumDegree</zorder>
   <zorder>maximumDegree</zorder>
   <zorder>distance</zorder>
   <zorder>area</zorder>
   <zorder>numberOfRegions</zorder>
   <zorder>length</zorder>
   <zorder>breadth</zorder>
   <zorder>fixedRegions</zorder>
   <zorder>comboBox</zorder>
   <zorder>measures</zorder>
   <zorder>bc</zorder>
   <zorder>dc</zorder>
   <zorder>ec</zorder>
   <zorder>cc</zorder>
   <zorder>numberOfSimulations</zorder>
   <zorder>label</zorder>
   <zorder>label_2</zorder>
   <zorder>label_3</zorder>
   <zorder>label_4</zorder>
   <zorder>label_5</zorder>
   <zorder>label_8</zorder>
   <zorder>label_9</zorder>
   <zorder>label_10</zorder>
   <zorder>label_11</zorder>
   <zorder>label_12</zorder>
   <zorder>label_13</zorder>
   <zorder>label_14</zorder>
   <zorder>label_15</zorder>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>beta</tabstop>
  <tabstop>alpha</tabstop>
  <tabstop>pushButton</tabstop>
  <tabstop>minimumDegree</tabstop>
  <tabstop>nodes</tabstop>
  <tabstop>maximumDegree</tabstop>
  <tabstop>distance</tabstop>
  <tabstop>area</tabstop>
  <tabstop>numberOfRegions</tabstop>
  <tabstop>length</tabstop>
  <tabstop>breadth</tabstop>
  <tabstop>fixedRegions</tabstop>
  <tabstop>fexibleRegions</tabstop>
  <tabstop>comboBox</tabstop>
  <tabstop>measures</tabstop>
  <tabstop>bc</tabstop>
  <tabstop>dc</tabstop>
  <tabstop>ec</tabstop>
  <tabstop>cc</tabstop>
  <tabstop>numberOfSimulations</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
